.github/workflows/tests.yaml

```diff
@@ -16,19 +16,30 @@ name: tests
 on: [pull_request]
 
 jobs:
-  build:
+  lint:
     name: lint
     runs-on: ubuntu-latest
     steps:
     - name: Setup Go
       uses: actions/setup-go@v2
       with:
-        go-version: '1.13'
+        go-version: '1.16'
     - name: Install goimports
       run: go get golang.org/x/tools/cmd/goimports
     - name: Checkout code
       uses: actions/checkout@v2
     - run: goimports -w .
     - run: go mod tidy
     - name: Verify no changes from goimports and go mod tidy. If you're reading this and the check has failed, run `goimports -w . && go mod tidy`.
-      run: git diff --exit-code
\ No newline at end of file
+      run: git diff --exit-code
+  version-mismatch:
```

Comment:
Should this just be part of link maybe? Then we don't have to do another checkout step and it's still just 1 status

Reply:
sure